README.md

```diff
@@ -59,9 +93,9 @@ In **PowerShell** or in a **Command Prompt** window:
         sqlite-devel.x86_64 readline-devel.x86_64 bzip2-devel.x86_64
     ```
 
-- **OS X**
+- **macOS**
 
-  Most problems on OS X have been with loading and linking OpenSSL and zlib. The following commands install necessary packages and tell the Python installer where to find them.
+  Most installation problems on macOS are related to loading and linking `OpenSSL` and `zlib`. The following command installs the necessary packages and tells the Python installer where to find them:
```

Comment:
The enclosing of OpenSSL within backticks is unnecessary.

Reply:
Will update. Should the `zlib` also have it's backticks removed?

Reply:
No, because it is being discussed in the capacity of a library.